bundles/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/internal/ComfoAirCommandType.java

```diff
@@ -0,0 +1,862 @@
+     *         ComfoAirCommand named by key
+     */
+    @SuppressWarnings("null")
+    public static Collection<ComfoAirCommand> getAffectedReadCommands(String key, Set<String> usedKeys) {
+
+        Map<Integer, ComfoAirCommand> commands = new HashMap<Integer, ComfoAirCommand>();
+
+        ComfoAirCommandType commandType = ComfoAirCommandType.getCommandTypeByKey(key);
+        if (commandType != null) {
+            if (commandType.read_reply_command != 0) {
+                Integer getCmd = commandType.read_command == 0 ? null : Integer.valueOf(commandType.read_command);
+                Integer replyCmd = Integer.valueOf(commandType.read_reply_command);
+
+                ComfoAirCommand command = new ComfoAirCommand(key, getCmd, replyCmd, new int[0], null, null);
+                commands.put(replyCmd, command);
+            }
+
+            if (commandType.change_affected != null) {
+                for (String affectedKey : commandType.change_affected) {
+                    // refresh affected event keys only when they are used
+                    if (!usedKeys.contains(affectedKey)) {
+                        continue;
+                    }
+
+                    ComfoAirCommandType affectedCommandType = ComfoAirCommandType.getCommandTypeByKey(affectedKey);
+
+                    if (affectedCommandType != null) {
+                        Integer getCmd = affectedCommandType.read_command == 0 ? null
+                                : Integer.valueOf(affectedCommandType.read_command);
+                        Integer replyCmd = Integer.valueOf(affectedCommandType.read_reply_command);
+
+                        ComfoAirCommand command = commands.get(replyCmd);
+
+                        if (command == null) {
+                            command = new ComfoAirCommand(affectedKey, getCmd, replyCmd, new int[0], null, null);
+                            commands.put(replyCmd, command);
+                        } else {
+                            command.addKey(affectedKey);
+                        }
+                    }
+                }
+            }
+        }
+        return commands.values();
+    }
+
+    /**
+     * Get all commands which receive informations to update items.
+     *
+     * @return all ComfoAirCommand's identified by keys
+     */
+    @SuppressWarnings("null")
+    public static Collection<ComfoAirCommand> getReadCommandsByEventTypes(List<String> keys) {
+
+        Map<Integer, ComfoAirCommand> commands = new HashMap<Integer, ComfoAirCommand>();
+        for (ComfoAirCommandType entry : values()) {
+            if (!keys.contains(entry.key)) {
+                continue;
+            }
+            if (entry.read_reply_command == 0) {
+                continue;
+            }
+
+            Integer getCmd = entry.read_command == 0 ? null : Integer.valueOf(entry.read_command);
+            Integer replyCmd = Integer.valueOf(entry.read_reply_command);
```

Comment:
Java automatically box/unboxes primitive types to their object equivalents so you don't need to do it manually.
```suggestion
            int getCmd = entry.read_command == 0 ? null : entry.read_command;
            int replyCmd = entry.read_reply_command;
```